### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (a[href$="classes"])

### ISSUE THREAD

**Issue by animania4ka** [ada-cat/accessibility#9]
**Issue**: `Elements must have sufficient color contrast` ([`color-contrast`](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))

**Target application**: [Ishinryu Karate SF](https://www.ishinryukaratesf.com/)

**Element path**: a[href$="classes"]

**Snippet**:

    <a href="/classes">Learn more</a>

**How to fix**:

    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 10.5pt, font weight: bold). Expected contrast ratio of 4.5:1

**Environment**:
Chrome version 75.0.3770.18

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
